.github/workflows/blank.yml

```diff
@@ -0,0 +1,27 @@
+name: Github Actions CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Docker version
+      run: docker version
+    - name: Debug if needed
+      run: |
+        [[ "$DEBUG" ]] && env
```

Comment:
What does it do? Is it something for actions to report more details? What value this supposed to have true/false? Also setting DEBUG flag as a secret feels odd and very unusual

Reply:
it can be useful for showing environment variables of Github Actions. This secret should any value . But I can delete this step if he seems confused

Reply:
that's fine to keep it. can be useful